library/src/webapp/js/portal/portal.theme.switcher.js

```diff
@@ -0,0 +1,71 @@
+var portal = portal || {};
```

Comment:
```suggestion
const portal = portal || {};
```

Reply:
Not sure if you can do that. portal is defined in includeStandardHead as a window scope object. I'll try though.

Reply:
I removed the portal bit. We now just define an object called allsites for scoping purposes.